src/Features/Blockcore.Features.Wallet/Database/WalletStore.cs

```diff
@@ -353,8 +378,11 @@ FROM TransactionData
 
         public TransactionOutputData GetForOutput(OutPoint outPoint)
         {
-            var trx = this.sqliteConnection.QueryFirstOrDefault<TransactionData>(
-                "SELECT * FROM TransactionData WHERE OutPoint = @outPoint", new { outPoint });
+            TransactionData trx = null;
+
+            using var conn = this.GetDbConnection();
+
```

Comment:
there is some inconsistency, sometimes using has a blank line, sometimes not (I prefer to not have blank line)